docs/how-to/deploy_sdcore_user_plane_in_dpdk_mode.md

```diff
@@ -1,23 +1,56 @@
 # Deploy SD-Core User Plane in DPDK mode
 
-This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using the `sdcore-user-plane-k8s` Terraform Module.
+This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using Terraform modules.
+Navigate between the tabs in each section of this guide to find the steps suitable for your setup (Kubernetes charm of Machine charm).
 
 ## Requirements
 
-- A Kubernetes cluster which meets below requirements:
-  - host CPU that supports AVX2, RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator or equivalent)
+``````{tab-set}
+
+`````{tab-item} Kubernetes Charm
```

Comment:
Instead of having many sections with their own tabs, couldn't we have the tab selected at the top level of the document and the rest of the page reflects the selection.

Reply:
Done.
The cost of this change however is that we won't get the `CONTENTS` panel generated (the one on the right).